flow/connectors/mysql/logger.go

```diff
@@ -0,0 +1,118 @@
+package connmysql
+
+import (
+	"fmt"
+	"os"
+
+	"go.temporal.io/sdk/log"
+)
+
+// implement interface at https://github.com/siddontang/go-log
+type BinlogLogger struct {
+	log.Logger
+}
+
+func (l BinlogLogger) Fatal(args ...any) {
+	l.Error(args...)
+	os.Exit(1)
+}
+
+func (l BinlogLogger) Fatalf(format string, args ...any) {
+	l.Errorf(format, args...)
+	os.Exit(1)
+}
+
+func (l BinlogLogger) Fatalln(args ...any) {
+	l.Errorln(args...)
+	os.Exit(1)
+}
+
+func (l BinlogLogger) Panic(args ...any) {
```

Comment:
do we want the ability for logging code to take down `flow-worker`?
seems like we can just make this an error

Reply:
This function should never be called, so it's a bit moot, but we have this issue with every dependency being able to call `panic` already. `Panic` is not supposed to return so the calling code will be in an irregular state afterwards anyways, may very well panic after that. Or do something worse